### DISCUSSION TITLE
Hands issue

### DISCUSSION THREAD

**Discussion by VolatileIvory** [VickyFrenzy/gmod-enhanced-camera-2#28] [Q&A] [answered]
Whenever I run with certain SWEPs (ArcCW pistols mostly) I can see both the view model hands and the 3rd person hands. Is there any way to fix this and if not can someone add an option to not see your arms?

**Comment by VickyFrenzy** [comment, marked as answer]
Hello! An issue with ArcCW weapons has already been opened here: #17
